net/net.sh

```diff
@@ -284,6 +297,8 @@ while getopts "h?T:t:o:f:rD:c:Fn:i:d" opt "${shortArgs[@]}"; do
   esac
 done
 
+grep -qE '^(on|off|auto|cuda)$' <<<"$gpuMode" || ( echo "Unexpected GPU mode: \"$gpuMode\"" && exit 1 )
```

Comment:
with `-g` gone, this check can move into the `--gpu-mode` if block above.  I'd prefer you use pure bash instead of forking off a grep